packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -3,14 +3,87 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:developer';
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
```

Comment:
Yea, I was playing around with WidgetBindings to see if I could quickly estimate time in platform channels via frame time instead of having to make a system call.  It doesn't work that way though unfortunately.